week1/intro.java

```diff
@@ -0,0 +1,38 @@
+class Foo {
+    int order = 1; 
```

Comment:
Should order be `volatile` because it's updated by multiple threads? (For liveness, I think this doesn't affect safety)